llvm/test/Transforms/FixIrreducible/switch.ll

```diff
@@ -1,5 +1,6 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN: opt < %s -passes='lower-switch,fix-irreducible' -S | FileCheck %s
+; RUN: opt < %s -lowerswitch -fix-irreducible --verify-loop-info -S | FileCheck %s
```

Comment:
Does this really need the old PM run lines?

Reply:
It doesn't really matter, but I have been copy-pasting existing lines, and kept this to be consistent with all the other tests.